apc_persist.c

```diff
@@ -521,17 +521,22 @@ static zend_bool apc_unpersist_serialized(
 	return 0;
 }
 
+/* Used to reduce hash collisions when using pointers in hash tables. (#175) */
+static zend_long apc_shr3(zend_ulong index) {
```

Comment:
Better return zend_ulong? That's both the input type and what the zend_hash functions accept.

Reply:
Fixed, that was a typo, I'd meant to type zend_ulong